src/app/forms/templates/psv/psv-tech-record.template.ts

```diff
@@ -46,7 +54,6 @@ export const PsvTechRecord: FormNode = {
           name: 'dtpNumber',
           label: 'DTP number',
           value: '',
```

Comment:
Could do with width adding for this box

![image](https://user-images.githubusercontent.com/92087051/196948085-62a3c52b-ddc8-41b4-aff9-0709915caa18.png)

Reply:
I left them out because they were out of scope and last I heard they wanted a dropdown for number of axles, IDK if that's true anymore

Reply:
Sorry I meant for dtpNumber